### PR TITLE
Promise rejection events sample

### PR DESCRIPTION
**Pull request by jeffposnick**
R: @domenic @robdodson @addyosmani @beaufortfrancois etc.

This is a code sample for [promise rejection events](https://www.chromestatus.com/feature/4805872211460096), added in M49.

You can view a live version at https://jeffy.info/samples/promise-rejection-events/

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
I'm not sure we need the overall complexity there with setInterval, maps, etc.
One button to create a rejected Promise and another button to create a handled rejected Promise seem more straightforward to simply log what is happening.

**Comment by jeffposnick** [comment]
Okay, fair enough. @beaufortfrancois PTAL.

**Comment by beaufortfrancois** [comment]
:+1:

File: promise-rejection-events/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  <code>rejectionhandled</code>, can be used to keep track of
+  <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise"><code>Promise</code></a>
+  rejections, including whether those rejections are handled after the fact.
+</p>
+<p>
+  The <code>unhandledrejection</code> event is the promise-equivalent of the
+  global <a href="https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onerror"><code>error</code></a>
+  event, which is fired for uncaught exceptions. Because a rejected promise
+  could be handled after the fact, by attaching <code>catch(onRejected)</code>
+  or <code>then(onResolved, onRejected)</code> to it, the additional
```
Comment (domenic) [inline promise-rejection-events/index.html:19]: s/onResolved/onFulfilled/

**Comment by domenic** [comment]
Awesome stuff, thanks for putting this together. Sorry for the delay in reviewing but I've been in standards meetings all week.

File: promise-rejection-events/index.html (hunk 1 above)
Comment (jeffposnick) [inline promise-rejection-events/index.html:19]: Righto; I'll patch the sample in a new PR.

**Pull request merged**